compiler/aarch64/codegen/ARM64Ops.hpp

```diff
@@ -0,0 +1,1225 @@
+		ARM64Op_cbnz_32_bit,                                            	/* 0x35000000	CBNZ      	 */
+		ARM64Op_cbz_64_bit,                                             	/* 0xB4000000	CBZ       	 */
+		ARM64Op_cbnz_64_bit,                                            	/* 0xB5000000	CBNZ      	 */
+	/* Test & branch (immediate) */
+		ARM64Op_tbz,                                                    	/* 0x36000000	TBZ       	 */
+		ARM64Op_tbnz,                                                   	/* 0x37000000	TBNZ      	 */
+	/* Conditional branch (immediate) */
+		ARM64Op_b_cond,                                                 	/* 0x54000000	B_cond    	 */
+	/* Exception generation */
+		ARM64Op_svc,                                                    	/* 0xD4000001	SVC       	 */
+		ARM64Op_hvc,                                                    	/* 0xD4000002	HVC       	 */
+		ARM64Op_smc,                                                    	/* 0xD4000003	SMC       	 */
+		ARM64Op_brk,                                                    	/* 0xD4200000	BRK       	 */
+		ARM64Op_hlt,                                                    	/* 0xD4400000	HLT       	 */
+		ARM64Op_dcps1,                                                  	/* 0xD4A00001	DCPS1     	 */
+		ARM64Op_dcps2,                                                  	/* 0xD4A00002	DCPS2     	 */
+		ARM64Op_dcps3,                                                  	/* 0xD4A00003	DCPS3     	 */
+	/* System */
+		ARM64Op_msr_immediate,                                          	/* 0xD500401F	MSR       	 */
+		ARM64Op_hint,                                                   	/* 0xD503201F	HINT      	 */
+		ARM64Op_clrex,                                                  	/* 0xD503305F	CLREX     	 */
+		ARM64Op_dsb,                                                    	/* 0xD503309F	DSB       	 */
+		ARM64Op_dmb,                                                    	/* 0xD50330BF	DMB       	 */
+		ARM64Op_isb,                                                    	/* 0xD50330DF	ISB       	 */
+		ARM64Op_sys,                                                    	/* 0xD5080000	SYS       	 */
+		ARM64Op_msr_register,                                           	/* 0xD5100000	MSR       	 */
+		ARM64Op_sysl,                                                   	/* 0xD5280000	SYSL      	 */
+		ARM64Op_mrs,                                                    	/* 0xD5300000	MRS       	 */
+	/* Unconditional branch (register) */
+		ARM64Op_br,                                                     	/* 0xD61F0000	BR        	 */
+		ARM64Op_blr,                                                    	/* 0xD63F0000	BLR       	 */
+		ARM64Op_ret,                                                    	/* 0xD65F0000	RET       	 */
+		ARM64Op_eret,                                                   	/* 0xD69F03E0	ERET      	 */
+		ARM64Op_drps,                                                   	/* 0xD6BF03E0	DRPS      	 */
```

Comment:
Ill have to dig into these. I haven't looked what they actually do. I think (from what I said in the other comment) that we should tag them unused in an enum, and add explanations rather than full removal, as it would leave traces and allow to add them back if needs be.